Extensions/Oxide.CSharp/PluginCompiler.cs

```diff
@@ -494,5 +494,34 @@ private static string GetHash(string filePath, HashAlgorithm algorithm)
                 return BitConverter.ToString(hash).Replace("-", string.Empty).ToLower();
             }
         }
+
+        private static bool FirewallBlocking()
+        {
+            try
+            {
+                // we can use any url below that contains a CSharp source file.
+                var request =
+                    (HttpWebRequest)WebRequest.Create(
+                        $"https://devdrop.blob.core.windows.net/downloads/FirewallTest.cs");
```

Comment:
I wouldn't use a random plugin from a site that isn't controlled by us, there are plugins available on https://github.com/umods such as DevTest, though I don't see why this would help test this at all.